app/components/Application/__tests__/index.js

```diff
@@ -3,8 +3,8 @@ import ReactTestUtils from 'react-addons-test-utils';
 import Application from '../index.jsx';
 import styles from '../style.sass';
 
-describe('Application', function() {
-  it('displays the component', function() {
+describe('Application', function () {
+  it('displays the component', function () {
```

Comment:
Sorry about that... fixed now through the awesome power of `npm run lint:js -- --fix` :)